### ISSUE TITLE
VS Code Source Control support for filesystem path using ssh://

### ISSUE THREAD

**Issue by csaska** [microsoft/vscode#109954]
#66046 was closed because Remote Development with VS Code was released as an alternative to SSH FS. This is not possible alternative to SSH FS for many users as ppc is not supported as detailed by microsoft/vscode-remote-release#401.

I woud like to see the VS Code Source Control feature support filesystem paths using ssh:// as originally described in #66046.

**Comment by manistal** [comment]
+1

**Comment by luisdatec** [comment]
Hey, FYI

I am using now [this extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh), and it let me access to the folder of the server, and also use the features of Git in my PC remotely.

**Comment by psolyca** [comment]
@luisdatec I was using this extension also but :

- it's not working with VSCodium
- if you look at your remote server, there are lots of extensions installed on the server side taking more and more space (this was the case 2 years ago).

The simple SSH from [@SchoofsKelvin ](https://github.com/SchoofsKelvin/vscode-sshfs)

**Comment by luisdatec** [comment]
Yes, I saw that they install a folder called `.vscode-server`, but at least work for Git. I was interested in that, because work in Windows is so annoying with the type of files, test, etc.

Let's wait for solution in the other extension.

**Comment by mitchelangelo** [comment]
+1
